multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/helpers/MtaDescriptorPropertiesResolver.java

```diff
@@ -120,6 +121,22 @@ private DeploymentDescriptor correctEntityNames(DeploymentDescriptor descriptor)
                       .validate();
     }
 
+    private void editRoutesAddLiveParameter(DeploymentDescriptor descriptor) {
```

Comment:
rename to something more descriptive:
replaceNullLiveRoutesWithEmpty or something